src/facade/command_id.h

```diff
@@ -93,7 +93,7 @@ class CommandId {
   static uint32_t OptCount(uint32_t mask);
 
  protected:
-  std::string_view name_;
+  std::string name_;
```

Comment:
curious, what caused the change?

Reply:
I accidentally tried to insert a string and it generated:

```
assert(PolicyTraits::apply(FindElement{*this}, *iterator_at(i)) == iterator_at(i) && "constructed value does not match the lookup key"
```

The string would go out of scope and make the `string_view` dangle. Basically it's error prone and I really don't see any value using non owning views on a class that's supposed to own it's members (e.g., string).